app/client/src/features/events/LiveFeedback/LiveFeedbackDialog.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { DialogContent } from '@material-ui/core';
+import { useMutation, graphql } from 'react-relay';
+
+import type { SubmitLiveFeedbackMutation } from '@local/__generated__/SubmitLiveFeedbackMutation.graphql';
+import { ResponsiveDialog } from '@local/components/ResponsiveDialog';
+import { LiveFeedbackForm, TLiveFeedbackFormState } from './LiveFeedbackForm';
+
+interface Props {
+    isOpen: boolean;
+    openDialog: string;
+    openLinked: () => void;
+    close: () => void;
+    eventId: string;
+}
+
+export const SUBMIT_LIVE_FEEDBACK_MUTATION = graphql`
+    mutation SubmitLiveFeedbackMutation($input: CreateFeedback!) {
```

Comment:
Be aware that when you change a file name with a graphql query/fragment/mutation/subscription that its name should be updated to match the filename. The compiler enforces this and you should see an error from the relay compiler like this when they don't match.
![Screenshot from 2022-02-01 01-48-40](https://user-images.githubusercontent.com/14946745/151946557-c20170be-52ba-41d9-a970-501028de27ea.png)